README.md

```diff
@@ -155,9 +155,17 @@ $ PGPASSWORD="password" psql -h 127.0.0.1 -U explorer databasename
 ```
 
 ## Deployments
-Both `indexer-explorer` and `circulating-supply` binaries are run within Docker, their `Dockerfile`s can be found within their respective directoires/workspaces. Docker images are built using Google Cloud Build and then deployed to Google Cloud Run. The following commands can be used to build the Docker images:
+Both `indexer-explorer` and `circulating-supply` binaries are run within Docker, their `Dockerfile`s can be found within their respective directories/workspaces. Docker images are built using Google Cloud Build and then deployed to Google Cloud Run. The following commands can be used to build the Docker images:
 
 ```bash
 $ docker build -f ./indexer/Dockerfile .
 $ docker build -f ./circulating-supply/Dockerfile .
 ```
+
+## Features
+The tables `account_changes` and `assets__fungible_token_events` can be enabled by [features](https://doc.rust-lang.org/cargo/reference/features.html) config in their `Cargo.toml`s files. Example:
```

Comment:
It should be possible just with adding flags:
```
cargo build --release --features "account_changes fungible_token_events"
```

Reply:
Sounds good, done